make_environment_gfortran

```diff
@@ -26,4 +33,4 @@ export   FC_MPI="mpifort"                                      # MPI
 export   FC_MPIFLAGS="${FC_FLAGS}"                             # Parallel code compiler flags
 export   FC_MPILD=${FC_LD}                                     # Serial linker flags
 # -------------------------------------------------------------------------------------------------------------------
-export   MPI_TMP="${HOME}/grasp_mpi_tmp"                       # Location for temporary files
+export   MPI_TMP="${HOME}/tmp"                       # Location for temporary files
```

Comment:
The PR needs to be brought up to speed with the latest master before merging. E.g. `make_environment_gfortran` is not a thing anymore. I don't think any of the substantial parts of the PR have conflicts though, just the build files.

Reply:
Yes, and the changes with copying blas and lapack libraries from a given folder are way to case specific. Better to set e.g. the LD_LIBRARY_PATH flag or let CMake do the job.